webapp/src/components/expanded_view/controls_button.tsx

```diff
@@ -125,3 +127,9 @@ const UnavailableIcon = styled.div`
 const TooltipSubtext = styled.div`
   opacity: 0.56;
 `;
+
+const StyledTooltip = styled(Tooltip)<{isDisabled?: boolean}>`
+  ${({isDisabled}) => isDisabled && css`
+      display: none;
+  `}
+`;
```

Comment:
We can put this is the `shared.tsx`

Reply:
Oh nice, I didn't realized we had a common file. Thanks :+1: